test/linter.js

```diff
@@ -4,23 +4,125 @@
 'use strict';
 var
   execSync = require('child_process').execSync,
-  utils = require('./util/utils');
+  utils = require('./util/utils'),
+  fs = require('fs'),
+  literate = require('@microsoft.azure/literate'),
+  async_io_1 = require("@microsoft.azure/async-io"),
+  path = require('path');
+
+async function readConfigFile(file, tag) {
```

Comment:
are the machines that will run this running new versions of JS? that support async/await?